### ISSUE TITLE
An error has been caught in function 'launch', process 'MainProcess' (941), thread 'MainThread' (139635913963328):

### ISSUE THREAD

**Issue by FlyingAnt2018** [Megvii-BaseDetection/YOLOX#1577]
i encounter this problem, when i start train a model no mater from pretrained model nor from scratch. how can i fix this problem?
![图片](https://user-images.githubusercontent.com/49147567/208397395-29131489-da01-4735-b9ba-339774308341.png)

**Comment by wudizuixiaosa** [comment]
hi，bro,I have the same problem as you,Is it solved now？

**Comment by FlyingAnt2018** [comment]
> 



> hi，bro,I have the same problem as you,Is it solved now？

yes, i have solve this problem by change batch_size to 2 or more, not 1

**Comment by wudizuixiaosa** [comment]
Thanks for your reply. My bz reports errors no matter 2, 4, 8 or 1，hh

**Comment by PranvQT** [comment]
Hi anyone was able to solve this? I tried changing batch size still same error..

**Comment by xiaochuanfa** [comment]
> Hi anyone was able to solve this? I tried changing batch size still same error..
Me too

**Comment by dauphongxd** [comment]
Check self.num_classes = #. If you have 2 classes, make sure you specific 2 classes.

**Comment by NaYeongHun** [comment]
downgrade numpy version
